docs/tutorials/projects.md

```diff
@@ -0,0 +1,347 @@
+- Lastly, note the top line "Cloning into 'hello-projects'..."; because this project is based out of a GitHub repository, it is _automatically_ portable to any remote location where both `git` and `prefect` are configured! You can convince yourself of this by either running a new local worker on a different machine, or by switching this deployment to run with your docker work pool (more on this shortly).
+
+!!! note "`prefect.yaml`"
+    The above process worked out-of-the-box because of the information stored within `prefect.yaml`; if you open this file up in a text editor, you'll find that is not empty.  Specifically, it contains the following `pull` step that was automatically populated when you first ran `prefect project init`:
+    ```yaml
+    pull:
+    - prefect.projects.steps.git_clone_project:
+        repository: https://github.com/PrefectHQ/hello-projects.git
+        branch: main
+    ```
+    These `pull` steps are the instructions sent to your worker's runtime environment that allow it to clone your project in remote locations. For more information, see [the project concept documentation](/concepts/projects/).
+
+
+### Dockerized deployment
+
+In this example, we extend the above two by dockerizing our setup and executing runs with a Docker Worker.  Building off the [git-based example above](#git-based-deployment), let's switch our deployment to submit work to the `docker-work` work pool that we started at the beginning:
+
+<div class="terminal">
+```bash
+$ prefect deploy -f 'log-flow' \
+    -n my-docker-git-deployment \
+    -p docker-work
+$ prefect deployment run 'log-flow/my-docker-git-deployment'
+```
+</div>
+
+As promised above, this worked out of the box!  
+
+Let's deploy a new flow from this project that requires additional dependencies that might not be available in the default image our work pool is using; this flow requires both `pandas` and `numpy` as a dependency, which we will also install locally first and confirm the flow is working:
+
+<div class="terminal">
+```bash
+$ pip install -r requirements.txt
+$ python flows/pandas_flow.py
+```
+</div>
+
+We now have two options for how to manage these dependencies in our worker's environment:
+
+- setting the `EXTRA_PIP_PACKAGES` environment variable or using another hook to install the dependencies at runtime
+- building a custom Docker image with the dependencies baked in
+
+The first option is the quickest, but it does run the risk of changing runtime state from one run to the next and is not recommended for production use cases.  For the second option, we need to configure a `build` step within our `prefect.yaml` file as follows (Note: if starting from scratch we could use the `docker-git` recipe):
```

Comment:
It felt like whiplash to be told about EXTRA_PIP_PACKAGES and then to be told it's not good.  Can we switch to just focus on what we're doing in the tutorial.  E.g.:

```suggestion
In this tutorial we will focus on building a custom Docker image. First, we need to configure a `build` step within our `prefect.yaml` file as follows (Note: if starting from scratch we could use the `docker-git` recipe):
```

Reply:
love that suggestion, yup